Missing 'elf_syms' and 'vmbuild' on clean build

>>> Done! IncludeOS bundle downloaded and installed


>>> Building chainloader
~/Documents/code/IncludeOS/src/chainload/build ~/Documents/code/IncludeOS
-- Building for arch i686, platform x86_nano
-- CMake root: /usr/local
-- Target CPU architecture i686
-- Target triple i686-pc-linux-elf
-- Using vanilla CPU features: SSE3. CAPABS = -msse3 -mfpmath=sse
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/jsolsvik/Documents/code/IncludeOS/src/chainload/build
[ 25%] Linking CXX executable chainloader
[100%] Built target service
make[2]: /usr/local/includeos/bin/elf_syms: No such file or directory
make[2]: /usr/local/includeos/bin/vmbuild: No such file or directory
make[2]: *** [CMakeFiles/pruned_elf_symbols] Error 1
make[1]: *** [CMakeFiles/pruned_elf_symbols.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/prepend_bootloader] Error 1
make[1]: *** [CMakeFiles/prepend_bootloader.dir/all] Error 2
make: *** [all] Error 2
>>> Sorry <<<
Could not build chainloader.

**Case**
I built IncludeOS normally, but with the wrong $INCLUDE_PREFIX. It led to filepaths beiing /usr/local/includeos/includeos/bin which led to errors. I could not run the ./test.sh successfully.

So I decided to do a "clean-buildl" whatever that is. I removed all contents of  the /usr/local/includeos/bin directory, including elf_syms and vmbuild. Then I tried to build agian, but now the build script is complaining that elf_syms and vmbuild is missing. I thought the install.sh would take care of these dependencies, as it did the first time I built IncludeOS. How to I do a completely clean build? I haven't found any guides on that.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,6 +141,7 @@ if(vmbuild)
   # Install vmbuilder as an external project
   ExternalProject_Add(vmbuild
     PREFIX vmbuild # Build where
+    BUILD_ALWAYS 1
     SOURCE_DIR ${INCLUDEOS_ROOT}/vmbuild # Where is project located
     BINARY_DIR ${INCLUDEOS_ROOT}/vmbuild/build
     INSTALL_DIR ${BIN} # Where to install
@@ -151,6 +152,7 @@ endif(vmbuild)
 option(diskbuilder "Build and install memdisk helper tool" ON)
 if(diskbuilder)
   ExternalProject_Add(diskbuilder
+    BUILD_ALWAYS 1
     SOURCE_DIR ${INCLUDEOS_ROOT}/diskimagebuild
     BINARY_DIR ${INCLUDEOS_ROOT}/diskimagebuild/build
     INSTALL_DIR ${BIN}

--- a/etc/install_dependencies_linux.sh
+++ b/etc/install_dependencies_linux.sh
@@ -7,7 +7,7 @@
 ############################################################
 
 BUILD_DEPENDENCIES="curl make cmake nasm bridge-utils qemu jq python-pip g++-multilib"
-CLANG_VERSION="3.8"
+[ ! -z "$CC" ] && { CLANG_VERSION=${CC: -3}; } || CLANG_VERSION="3.8"
 TEST_DEPENDENCIES="g++"
 PYTHON_DEPENDENCIES="jsonschema psutil junit-xml filemagic"
 INSTALLED_PIP=0

